### DISCUSSION TITLE
正反选图标展示不完整

### DISCUSSION THREAD

**Discussion by leon9508** [antvis/G2#5659] [Q&A]
- **G2 Version**: 4.2.10
- **Platform**: chrome
- **Mini Showcase(like screenshots)**: 
- 
![image](https://github.com/antvis/G2/assets/25746834/97108634-9ba3-4168-a536-760aec1e3564)

- **CodePen Link**:

存在分页的情况下，正反选图标显示不完整，上下边缘被横切。详见 ：
https://g2-v4.antv.vision/zh/examples/other/big-data#big-data-line（代码编辑区增加   chart.legend({radio: {}})  后 可见）

**Comment by pearmini** [comment]
可以尝试减下 radio 或者移动图例

**Reply by leon9508** [reply]
无法调整radio大小，移动图例试了也不能避免
